### PR TITLE
Update linux-riscv64 jdk11u pipeline to temurin

### PR DESCRIPTION
**Pull request by luhenry**
The backport has been merged upstream with [1].

[1] https://github.com/openjdk/riscv-port-jdk11u/commit/309291f166821443393818beed364de6f2de5c52

### CODE REVIEW

**Comment by luhenry** [comment]
Depends on https://github.com/adoptium/temurin-build/issues/3677

File: pipelines/jobs/configurations/jdk11u_pipeline_config.groovy [hunk 1]
```diff
@@ -183,29 +183,35 @@ class Config11 {
...
                 ],
                 crossCompile         : [
                         'hotspot'    : 'dockerhost-rise-ubuntu2204-aarch64-1',
+                        'temurin'    : 'dockerhost-rise-ubuntu2204-aarch64-1',
                         'openj9'     : 'x64',
                         'bisheng'    : 'x64'
                 ],
                 buildArgs            : [
                         'hotspot'    : '--create-sbom',
+                        'temurin'    : '--create-sbom',
```
Comment (sxa) [inline pipelines/jobs/configurations/jdk11u_pipeline_config.groovy:202]: Should we remove the hotspot line from these while replacing with Temurin?

**Review by sxa — APPROVED** [review]
Just one thought but not critical that it is addressed

File: pipelines/jobs/configurations/jdk11u_pipeline_config.groovy (hunk 1 above)
Comment (luhenry) [reply pipelines/jobs/configurations/jdk11u_pipeline_config.groovy:202]: I wasn't sure what the policy should be, happy to go either way

Comment (luhenry) [reply pipelines/jobs/configurations/jdk11u_pipeline_config.groovy:202]: I will remove then

**Pull request merged**